tests/main/disk-space-awareness/task.yaml

```diff
@@ -0,0 +1,72 @@
+summary: Check that basic snap operation are aware of low disk space.
+
+# this test is remounts /var/lib/snapd which is too intrusive for core and
+# selinux, disabled on these systems.
```

Comment:
I think you can relabel to run this test on selinux-enabled systems. CC @bboozzoo in case I'm mistaken.

Reply:
Yes, this is almost certainly possible, I just wanted to avoid complications here and have a basic test first. I can explore this in a followup.